internal/pkg/aws/cloudformation/cloudformation.go

```diff
@@ -218,6 +218,19 @@ func (c *CloudFormation) TemplateBodyFromChangeSet(changeSetID, stackName string
 	return aws.StringValue(out.TemplateBody), nil
 }
 
+// Outputs returns the outputs of a stack description.
+func (c *CloudFormation) Outputs(stack *Stack) (map[string]string, error) {
+	stackDescription, err := c.Describe(stack.Name)
+	if err != nil {
+		return nil, fmt.Errorf("retrieve outputs of stack description: %w", err)
+	}
+	outputs := make(map[string]string)
+	for _, output := range stackDescription.Outputs {
+		outputs[*output.OutputKey] = *output.OutputValue
```

Comment:
Can we use `aws.StringValue` here in the unlikely event that the SDK barfs and we have a nil pointer dereference?

Reply:
Done!